open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -39,7 +39,7 @@
 from hackathon.database import User, AdminHackathonRel, DockerHostServer, HackathonLike, \
```

Comment:
Please also remove those imports if it's not used any more which try to import MySQL models